### PR TITLE
Refine error information in userptr ioctl

### PR DESCRIPTION
**Pull request by mamin506**
Better error information when userptr failed.

### CODE REVIEW

File: src/runtime_src/driver/zynq/drm/zocl/zocl_bo.c [hunk 1]
```diff
@@ -276,15 +276,19 @@ zocl_userptr_bo_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
...
-	if (args->flags & XCL_BO_FLAGS_EXECBUF)
+	if (args->flags & XCL_BO_FLAGS_EXECBUF) {
+		DRM_ERROR("Exec buf could not be a user buffer\n");
 		return -EINVAL;
+	}
 
 	bo = zocl_create_userprt_bo(dev, args->size);
 	if (IS_ERR(bo)) {
-		DRM_DEBUG("object creation failed\n");
+		DRM_INFO("Object creation failed\n");
```
Comment (larry9523) [inline src/runtime_src/driver/zynq/drm/zocl/zocl_bo.c]: Why DRM_INFO is used here instead of DRM_ERROR?

**Comment by sonals** [comment]
ok to test

**Comment by NaAl-Xlnx** [comment]
retest this please

File: src/runtime_src/driver/zynq/drm/zocl/zocl_bo.c (hunk 1 above)
Comment (mamin506) [reply src/runtime_src/driver/zynq/drm/zocl/zocl_bo.c]: Yes. DDR_ERROR should be used.

**Comment by mamin506** [comment]
Please test again

**Comment by maxzhen** [comment]
retest this please

**Pull request merged**